samples/BasicApp/src/BasicApp.cpp

```diff
@@ -36,6 +36,7 @@
 */
 
 #include "cinder/app/App.h"
+#include "cinder/gl/gl.h"
 #include "cinder/gl/Pbo.h"
```

Comment:
Fyi, gl.h is now an umbrella header, so all other cinder/gl includes can be removed.